bibliopixel/main/all_pixel_test.py

```diff
@@ -12,10 +12,10 @@
 DESCRIPTION = """
 Equivalent to
 
-::
-
-    bp --num=10 --loglevel=debug --animation=strip_test --driver=serial \\
-       --fail_on_exception --layout=strip --ledtype=<argument>
+``````
```

Comment:
Is your keyboard skipping or what?  :-D

OK, all kidding aside, you must have some reason for having six of these...!

You are using this consistently.  But it's horrific.  I mean, what happens if you get 5 or 7 of these?

What's wrong with `::`?

Reply:
I have no idea what happened there. Will fix.

Reply:
Oh oh... that's required just like in commands.py
You have to escape the ` and the way to do that is ``
But markdown needs ```, so it becomes ``````